files/rabbitmq-backup

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+DEST="$1"
+python /var/lib/rabbitmq/mnesia/*plugins-expand/rabbitmq_management-*/priv/www/cli/rabbitmqadmin export $DEST -c /etc/rabbitmqadmin.conf -N host_ssl -q
```

Comment:
This is a rather weird place for the admin script.  The [documented way to get the script](https://www.rabbitmq.com/management-cli.html) seems to be to download it from https://rabbitmq.net.opencraft.hosting:15671/cli/, so maybe we should do that in the Ansible playbook and store it in `/usr/local/sbin`.  However, I can't access this URL at the moment (the connection times out when I try).